src/test/java/com/gojek/beast/sink/bq/BaseBQTest.java

```diff
@@ -81,7 +83,7 @@ protected TestMessage getTestMessage(String id, Instant now) {
     protected List<Record> getKafkaConsumerRecords(ColumnMapping columnMapping, Instant now, String topicName,
                                                    int partitionStart, long offsetStart, Clock clock, TestMessage...testMessages) throws InvalidProtocolBufferException {
         ProtoParser protoParser = new ProtoParser(StencilClientFactory.getClient(), TestMessage.class.getName());
-        ConsumerRecordConverter customConverter = new ConsumerRecordConverter(new RowMapper(columnMapping), protoParser, clock);
+        ConsumerRecordConverter customConverter = new ConsumerRecordConverter(new RowMapper(columnMapping), protoParser, clock, ConfigFactory.create(AppConfig.class, System.getProperties()));
```

Comment:
Lets not create appConfig if its not being used.

Reply:
It is getting used on line number 40.